openstack/resource_openstack_compute_quotaset_v2.go

```diff
@@ -0,0 +1,325 @@
+package openstack
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/extensions/quotasets"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceComputeQuotasetV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeQuotasetV2Create,
+		Read:   resourceComputeQuotasetV2Read,
+		Update: resourceComputeQuotasetV2Update,
+		Delete: resourceComputeQuotasetV2Delete,
```

Comment:
you can also use this func:
```suggestion
		Delete: schema.RemoveFromState,
```

Reply:
Nice catch